md/release-notes.md

```diff
@@ -26,14 +151,4 @@ Before you start Bonita Studio 7.8 or migrate your production to Bonita 7.8., ma
 It is no longer possible to import BAR files created with Bonita 5.x in the Studio. If you still need to migrate 5.x bar, use Bonita Studio __7.7.x__ version.
```

Comment:
Maybe we should add that the "resources/forms" directory of the business archives gets removed from the database when migartion to 7.8.x

Reply:
ok after explanation by Romain (!). Added.